src/it/MCHECKSTYLE-129/pom.xml

```diff
@@ -57,7 +57,7 @@ under the License.
         <artifactId>maven-checkstyle-plugin</artifactId>
         <version>${checkstyleVersion}</version>
         <configuration>
-          <configLocation>http://svn.apache.org/repos/asf/maven/plugins/tags/maven-checkstyle-plugin-2.15/src/main/resources/config/sun_checks.xml</configLocation>
+          <configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/main/resources/sun_checks.xml</configLocation>
```

Comment:
I don't recommend using master as it is the development line of the configuration. If we add a breaking compatibility in it, your release version of checkstyle could fail.
You should stick to a tag or release.

Reply:
@rnveach You are totally right. Good catch !
I had thought about this problem but I forgot to change the link